cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -48,7 +49,8 @@ var (
 	master               = flag.String("master", "", "Master URL to build a client config from. Either this or kubeconfig needs to be set if the provisioner is being run out of cluster.")
 	kubeconfig           = flag.String("kubeconfig", "", "Absolute path to the kubeconfig file. Either this or master needs to be set if the provisioner is being run out of cluster.")
 	csiEndpoint          = flag.String("csi-address", "/run/csi/socket", "The gRPC endpoint for Target CSI Volume.")
-	connectionTimeout    = flag.Duration("connection-timeout", 0, "This option is deprecated.")
+	_                    = deprecatedflags.Add("connection-timeout")
+	_                    = deprecatedflags.AddBool("foo")
```

Comment:
remove testing arg (I assume thats what this is)

Reply:
Oops, forgot to force-push. I had fixed that already.